aggregator/agglayer_client.go

```diff
@@ -0,0 +1,81 @@
+package aggregator
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/0xPolygon/cdk-rpc/types"
+	"github.com/0xPolygonHermez/zkevm-aggregator/rpclient"
```

Comment:
Should we switch to https://github.com/0xPolygon/cdk-rpc/blob/main/rpc/client.go ?

Reply:
Fix: https://github.com/0xPolygonHermez/zkevm-aggregator/pull/87/commits/e7283b5ff07b0c34cee31743aea3a1a5390a6b0f